src/tbb/cache_aligned_allocator.cpp

```diff
@@ -121,7 +131,11 @@ void initialize_handler_pointers() {
         FreeHandler = &std::free;
         MallocHandler = &std::malloc;
         padded_allocate_handler = &padded_allocate;
+#if !defined(__TBB_USE_POSIX_MEMALIGN)
```

Comment:
Usually if we have `else` branch in conditions we put the positive branch first.

Reply:
That's ok. I usually list the "more preferred" or "more usual" cases first to ease code reading. But I can swap the branches.